x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/indicator_match_rule_suppression_ess_basic.cy.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNewThreatIndicatorRule } from '../../../../objects/rule';
+
+import {
+  ALERT_SUPPRESSION_FIELDS_INPUT,
+  ALERT_SUPPRESSION_FIELDS,
+} from '../../../../screens/create_new_rule';
+import {
+  SUPPRESS_FOR_DETAILS,
+  DETAILS_TITLE,
+  SUPPRESS_BY_DETAILS,
+  SUPPRESS_MISSING_FIELD,
+  DEFINITION_DETAILS,
+  ALERT_SUPPRESSION_INSUFFICIENT_LICENSING_ICON,
+} from '../../../../screens/rule_details';
+
+import { selectIndicatorMatchType } from '../../../../tasks/create_new_rule';
+import { startBasicLicense } from '../../../../tasks/api_calls/licensing';
+import { createRule } from '../../../../tasks/api_calls/rules';
+import { login } from '../../../../tasks/login';
+import { visit } from '../../../../tasks/navigation';
+import { CREATE_RULE_URL } from '../../../../urls/navigation';
+import { TOOLTIP } from '../../../../screens/common';
+import { ruleDetailsUrl } from '../../../../urls/rule_details';
+import { getDetails } from '../../../../tasks/rule_details';
+
+const SUPPRESS_BY_FIELDS = ['myhash.mysha256', 'source.ip.keyword'];
+
+describe(
+  'Detection rules, Indicator Match, Alert Suppression',
+  {
+    tags: ['@ess'],
+  },
+  () => {
+    describe('Create rule form', () => {
+      beforeEach(() => {
+        login();
+        visit(CREATE_RULE_URL);
+        startBasicLicense();
+      });
+
+      it('can not create rule with rule execution suppression on basic license', () => {
+        selectIndicatorMatchType();
+
+        cy.get(ALERT_SUPPRESSION_FIELDS_INPUT).should('be.disabled');
+        cy.get(ALERT_SUPPRESSION_FIELDS).trigger('mouseover');
```

Comment:
NIT: If you are trying to perform a hover action in that element, `realHover();` might be a better option since for what we know should be less flaky.

Reply:
I tried to use `.realHover()`, but it did not work. Tests are failing. So, I leaving it with `.trigger('mouseover');`

Reply:
Sounds good to me, thanks! :)